rpc/client.gen.go

```diff
@@ -0,0 +1,546 @@
+// Package rpc provides primitives to interact with the openapi HTTP API.
```

Comment:
@Jasonyou1995 Should these files be introduced?

Reply:
These files are not necessary, I didn't know why they showed up.